catalog.tests.bom

```diff
@@ -0,0 +1,126 @@
+  - id: test-new-volume-customizer-openstack-vcd
+    version: "0.5.0-SNAPSHOT" # BROOKLYN_BLOCKSTORE_VERSION
+    itemType: template
+    name: New Volume Customizer tests for OpenStack and vCD
+    description: Attaching additional disks in provisioning time and after provisioning via effector is running correctly
+    item:
+      services:
+      - type: new-volume-customizer-openstack-vcd
+
+      - type: test-case
+        brooklyn.config:
+          targetId: parent-app
+          targetResolutionTimeout: 10s
+          timeout: 20m
+
+        brooklyn.children:
+
+        - type: assert-up-and-running-initial
+          name: "1. Node up and running"
+
+        - type: test-ssh
+          name: "2. Additional mount devices successfully added"
+          brooklyn.config:
+            command: df -h | grep /mount/brooklyn/b
+            assert.out:
+              contains: /mount/brooklyn/b
+            command: df -h | grep /mount/brooklyn/c
+            assert.out:
+              contains: /mount/brooklyn/c
+            command: egrep "/mount/brooklyn/b" /proc/mounts
+            assert.out:
+              contains: rw
+            command: egrep "/mount/brooklyn/c" /proc/mounts
+            assert.out:
+              contains: rw
+
+        - type: invoke-effector
+          name: "3. Invoke addExtraHdd effector"
+          effector: addExtraHdd
+          params:
+            location.customizer.fields:
+              blockDevice:
+                sizeInGb: 1
+                deviceSuffix: 'd'
+                deleteOnTermination: true
+                tags:
+                  brooklyn: "br-example-test-1"
+              filesystem:
+                mountPoint: "/mount/brooklyn/d"
+                filesystemType: "ext3"
+
+
+        - type: test-ssh
+          name: "4. addExtraHdd effector executed successfully"
+          brooklyn.config:
+            command: df -h | grep /mount/brooklyn/d
+            assert.out:
+              contains: /mount/brooklyn/d
+            command: egrep "/mount/brooklyn/d" /proc/mounts
+            assert.out:
+              contains: rw
+
+
+  - id: test-new-volume-customizer-aws
+    version: "0.5.0-SNAPSHOT" # BROOKLYN_BLOCKSTORE_VERSION
+    itemType: template
+    name: New Volume Customizer tests for AWS
+    description: Attaching additional disks in provisioning time and after provisioning via effector is running correctly
+    item:
+      services:
+      - type: new-volume-customizer-aws
+
+      - type: test-case
+        brooklyn.config:
+          targetId: parent-app
+          targetResolutionTimeout: 10s
+          timeout: 20m
+
+        brooklyn.children:
+
+        - type: assert-up-and-running-initial
+          name: "1. Node up and running"
+
+        - type: test-ssh
+          name: "2. Additional mount devices successfully added"
+          brooklyn.config:
+            command: df -h | grep /mount/brooklyn/h
```

Comment:
Invalid YAML ... duplicate keys

I think you should combine the multiple commands into 
```YAML
type: test-ssh
  name: "2. Additional mount devices successfully added"
  brooklyn.config:
    command: |
      set -e
      df -h | grep /mount/brooklyn/h
      check 2
      check 3
```